bindings/zeebe/command/command.go

```diff
@@ -30,7 +30,7 @@ import (
 const (
 	// operations.
 	TopologyOperation         bindings.OperationKind = "topology"
-	DeployProcessOperation    bindings.OperationKind = "deploy-process"
```

Comment:
If you want to change this, I'm ok, but `deploy-process` must be kept as an alias for backwars compatibility please